### PR TITLE
[WIP] make it possible to enable watchOptions poll for webpack

### PR DESCRIPTION
**Pull request by bjarneo**
Polling is the solution whenever you want a VM to notice changes made outside that VM.

### CODE REVIEW

File: packages/roc-package-webpack-web-dev/src/config/roc.config.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const config = {
         dev: {
             devMiddleware: {
                 noInfo: true,
-                quiet: false
+                quiet: false,
+                poll: false
```
Comment (dlmr) [inline packages/roc-package-webpack-web-dev/src/config/roc.config.js]: Should we maybe support defining a custom interval, not just the default?
https://github.com/webpack/watchpack#api

Have we made any performance tests with this, maybe we need to also use `aggregateTimeout`?

**Comment by dlmr** [comment]
Looks good! Will try this out!

This might also fix the problem we have seen with Dropbox setups.

File: packages/roc-package-webpack-web-dev/src/config/roc.config.js (hunk 1 above)
Comment (bjarneo) [inline packages/roc-package-webpack-web-dev/src/config/roc.config.js]: I'll update the PR with support for interval. 

And I'll take a look at aggregateTimout.

**Comment by bjarneo** [comment]
Updating this PR to a WIP

**Comment by dlmr** [comment]
This has now been implemented in #10

**Pull request closed without merging**